envoy/cilium_l7policy.cc

```diff
@@ -73,27 +73,34 @@ createPolicyMap(Server::Configuration::FactoryContext& context) {
 
 } // namespace
 
-Config::Config(const std::string& policy_name, const std::string& access_log_path,
+Config::Config(const std::string& policy_name, const std::string& access_log_path, const std::string& denied_403_body,
```

Comment:
Line too long. Needs a pass of Envoy format fixing?

Reply:
Fixed. Will need to fix formatting overall later, don't want that much churn in the sources right now...